core/src/main/java/com/digitalpebble/stormcrawler/bolt/FetcherBolt.java

```diff
@@ -402,6 +404,8 @@ public FetcherThread(Config conf, int num) {
 
             this.maxCrawlDelay = ConfUtils.getInt(conf,
                     "fetcher.max.crawl.delay", 30) * 1000;
+            this.crawlDelay = (long) (ConfUtils.getFloat(conf,
```

Comment:
probably don't need to read this but instead use fiq.crawlDelay as you do in line 534